flink-connectors/flink-connector-kafka-0.10/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumer010.java

```diff
@@ -259,4 +272,25 @@ protected AbstractPartitionDiscoverer createPartitionDiscoverer(
 		consumer.close();
 		return result;
 	}
+
+	/**
+	 * If ratelimiting is enabled, set maxBytesPerSecond per consumer based on a global ratelimit.
+	 * @param runtimeContext
+	 * @param properties
+	 */
+	public void setRateLimitingProperties(
```

Comment:
Move all of this logic to RateLimitingConfig.

Reply:
Done.